index.bs

```diff
@@ -1103,7 +1103,7 @@ optional |expires|,
 |sameSite|,
 run the following steps:
 
-1. If |name|, |value|, |domain|, or |path| contain U+003B (`;`), any [=C0 control=] character, or U+007F, then return failure.
+1. If |name| or |value| contain U+003B (`;`), any [=C0 control=] character except U+0009 (the horizontal tab character), or U+007F, then return failure.
```

Comment:
What happens to validation for domain and path?

Reply:
I added a comment here about this - https://github.com/httpwg/http-extensions/issues/1593#issuecomment-895385796

In the RFC, the character restrictions apply to attribute values derived from Set-Cookie headers, but in the `Storage Model` section it only says that these character checks should happen for `name` and `value`...  I think this may be more of an oversight than intentional, but regardless, my thinking with the change above is that this spec should match what's in the RFC today and then when/if the character restrictions are made to apply to attribute values as well, we can update it here also. What do you think?

Reply:
SGTM, but I'd add a `TODO: ...` or `Issue: ...` block to the spec referencing the issue.

Reply:
With what I added with the latest commit, it looks like this:

![image](https://user-images.githubusercontent.com/36489577/129422494-75b19d81-014b-4e95-a102-e8620c063f6e.png)

Does that seem reasonable?

Reply:
LGTM. Since the _expires_ and _sameSite_ attributes aren't passed through from script as strings I don't think those are a concern.